templates/elixir/flake.nix

```
@@ -0,0 +1,43 @@
+{
+  description = "Elixir template";
+
+  inputs.capacitor.url = "git+ssh://git@github.com/flox/capacitor?ref=v0";
+  inputs.capacitor.inputs.root.follows = "/";
+  inputs.capacitor.inputs.nixpkgs.follows = "nixpkgs/nixpkgs-stable";
+
+  inputs.nixpkgs.url = "git+ssh://git@github.com/flox/nixpkgs-flox";
+  inputs.nixpkgs.inputs.capacitor.follows = "capacitor";
+
+  inputs.floxpkgs.url = "git+ssh://git@github.com/flox/floxpkgs";
+  inputs.floxpkgs.inputs.capacitor.follows = "capacitor";
+  inputs.floxpkgs.inputs.nixpkgs.follows = "nixpkgs";
+
+  nixConfig.bash-prompt = "[flox]\\e\[38;5;172mλ \\e\[m";
```

I just merged https://github.com/flox/floxpkgs/pull/39 which does: 
```suggestion
  nixConfig.bash-prompt = "[flox] \\[\\033[38;5;172m\\]λ \\[\\033[0m\\]";
```